tools/mo/openvino/tools/mo/pipeline/common.py

```diff
@@ -173,16 +173,14 @@ def convert_inputs_of_specific_ops(graph: Graph):
 
 def prepare_emit_ir(graph: Graph, data_type: str, output_dir: str, output_model_name: str,
                     mean_data: [list, None] = None, input_names: list = None, meta_info: dict = None,
-                    use_temporary_path=False, used_by_ir_reader=False):
+                    use_temporary_path=False, convert_types=False):
     if input_names is None:
         input_names = []
     if meta_info is None:
         meta_info = {}
     graph.strict_mode = False
 
-    # temporary disable new FP16 generation
-    # if not used_by_ir_reader:
-    if True:
+    if convert_types:
```

Comment:
why do we need to change `Parameter` type here?

Reply:
This is done only if convert_types is True. Initially we agreed to not remove code that does old fp16 IR generation, lets do it in separate PR if we want to remove it completely.